modules/cli/pages/cbstats/cbstats-collections.adoc

```diff
@@ -64,65 +192,63 @@ Each data key is concluded with a colon, and the associated value is presented t
 
 Within the output, each successive group of lines presents information on one of the collections for the specified bucket, on the specified node.
 
-The first collection is identified, on each of the first eight lines, as `0x0:0x0`; and, on line four, as the `&#95;default` collection.
-Line one of the output displays the total disk space occupied by this collection for the specified bucket, on the specified node: this total (here, `19827828`) is expressed in _bytes_.
-The second line displays the number of items within the collection (here, `31592`).
-The third line displays, again in _bytes_, the total memory used by this collection (here, `21760360`).
-Lines five to seven display the numbers of deletions, gets, and writes for documents within this collection (which are `0`, `2`, and `2`, respectively).
-The eighth line identifies the scope within which this collection resides: in this case, the scope is the `&#95;default` scope.
+The first collection is identified, on each of the first nine lines, as `0x0:0x0`; and, on line five, as the `&#95;default` collection.
+
+Line one displays, in _bytes_, the total memory used by this collection (here, `20962645`).
+Line two displays the total disk space occupied by this collection for the specified bucket, on the specified node: this total (here, `2820666`) is, again, expressed in _bytes_.
+Line three indicates whether changes made to documents in the collection are _captured_, so that a historical record is maintained: if the value is `true`, a record is being maintained (note that the value can only be `false` for the `&#95;default` collection; and can only be `true` for any other collection when _Magma_ has been configured as the storage engine for the bucket).
```

Comment:
In this line, default collection can now have history enabled.

Reply:
Ok.

Reply:
I've changed to: Line three indicates whether changes made to documents in the collection are captured, so that a historical record is maintained: if the value is true, a record is being maintained (note that the value can only be true when Magma has been configured as the storage engine for the bucket).